src/index.js

```diff
@@ -25,6 +25,20 @@ function getEventHandlerOptions(instance, eventName) {
   return handlerOptions;
 }
 
+// borrowed from react-is
+const hasSymbol = typeof Symbol === 'function' && Symbol.for;
+const REACT_FORWARD_REF_TYPE = hasSymbol ? Symbol.for('react.forward_ref') : 0xead0;
```

Comment:
This is not necessary. Every in-support browser [supports `Symbol`](https://caniuse.com/#search=Symbol). People who need to support dead browsers are responsible for polyfilling their own site.

Reply:
I was attempting to duplicate the relevant code from `react-is`. I would have just pulled in the package as a dependency or peer dependency but I had difficulty figuring out your build system.

If you like I can simplify and remove the checks, and assume that symbols were available when react was loaded, but this is a one time check at load time.

Reply:
`Symbol` is a JavaScript language feature, not a React feature, so there's no need to perform this check: we can trust they exist.

Reply:
To clarify, React itself uses different brands depending on if Symbol exists. The code to check the brand depends on whether Symbol existed at the time React was loaded. If you don't want to support pre-symbol browsers, I can remove the  magic number and always assume the brand is a Symbol.